zebra-chain/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 
 [features]
 default = []
-proptest-impl = ["proptest", "proptest-derive", "itertools"]
+proptest-impl = ["proptest", "proptest-derive", "itertools", "zebra-test"]
```

Comment:
Follow-Up: Can you open a short ticket to rename proptest-impl to something more generic?

This change isn't a high priority right now. And it risks disabling some tests if we've hard-coded the feature name in our CI workflows. So I think it's best left for later.

Reply:
Done: #2424